src/main/java/mate/academy/dao/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
+    private static final String USER_NAME = "root";
+    private static final String PASSWORD = "bodyaA20041@";
+    private static final String DRIVER_NAME = "com.mysql.cj.jdbc.Driver";
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.setProperty("user", USER_NAME);
+        DB_PROPERTIES.setProperty("password", PASSWORD);
+        try {
+            Class.forName(DRIVER_NAME);
+        } catch (ClassNotFoundException e) {
+            throw new DataProcessingException("Can`t load JDBC driver", e);
```

Comment:
Use a more informative message for the exception. For example, 'Can't load JDBC driver com.mysql.cj.jdbc.Driver'.